src/components/EditCoursePage/CollapsibleCourseRunFields.jsx

```diff
@@ -47,136 +47,119 @@ const formatCourseRunTitle = (courseRun) => {
-                </React.Fragment>
-              }
-              placeholder="mm/dd/yyyy"
-              required
-            />
-            <Field
-              name={`${courseRun}.end`}
-              type="date"
-              component={RenderInputTextField}
-              format={value => getDateString(value)}
-              label={
-                <React.Fragment>
-                  End date
-                  <span className="required" aria-hidden>*</span>
-                </React.Fragment>
-              }
-              placeholder="mm/dd/yyyy"
-              required
-            />
-            <Field
-              name={`${courseRun}.min_effort`}
-              type="number"
-              component={RenderInputTextField}
-              label={
-                <React.Fragment>
-                  Minimum Effort
-                </React.Fragment>
-              }
-            />
-            <Field
-              name={`${courseRun}.max_effort`}
-              type="number"
-              component={RenderInputTextField}
-              label={
-                <React.Fragment>
-                  Maximum Effort
-                </React.Fragment>
-              }
-            />
-            <Field
-              name={`${courseRun}.pacing_type`}
-              type="text"
-              component={RenderSelectField}
-              options={pacingTypeOptions}
-              label={
-                <React.Fragment>
-                  Course Pacing
-                </React.Fragment>
-              }
-            />
-            <Field
-              name={`${courseRun}.content_language`}
-              type="text"
-              component={RenderSelectField}
-              options={languageOptions}
-              label={
-                <React.Fragment>
-                  Content Language
-                </React.Fragment>
-              }
-            />
-            <div className="transcript-label">
-              Transcript Languages
-            </div>
-            <FieldArray
-              name={`${courseRun}.transcript_languages`}
-              component={TranscriptLanguage}
-              languageOptions={languageOptions}
-            />
-            <Field
-              name={`${courseRun}.weeks_to_complete`}
-              type="number"
-              component={RenderInputTextField}
-              label={
-                <React.Fragment>
-                  Length
-                </React.Fragment>
-              }
-            />
-            <Field
-              name={`${courseRun}.staff`}
-              component={StaffList}
-            />
-          </Collapsible>
-        ))}
-        <button
-          type="button"
-          className="btn btn-light js-add-button"
-          onClick={() => fields.push({})}
+const CollapsibleCourseRunFields = (props) => {
+  const {
+    fields,
+    courseRuns,
+    languageOptions,
+    pacingTypeOptions,
+  } = props;
```

Comment:
Nit: Seems a little cleaner to destructure the props right when they're passed in like https://github.com/edx/publisher-frontend/blob/d57ff232fd11713e8ef35e1e3093d8b0f92c017c/src/components/RemoveButton/index.jsx#L5-L10